library/src/main/java/com/github/pwittchen/swipe/library/rx2/SimpleSwipeListener.java

```diff
@@ -0,0 +1,37 @@
+package com.github.pwittchen.swipe.library.rx2;
+
+import android.view.MotionEvent;
+
+public abstract class SimpleSwipeListener implements SwipeListener {
```

Comment:
This class is unused, but I see the point of keeping it in library project in case we want to consume single event or several events only, so let's keep it.